docs/plugins/yamleditor.rst

```diff
@@ -0,0 +1,84 @@
+Yamleditor Plugin
+=================
+The ``yamleditor`` plugin lets you open the tags, fields from a group of items, edit them in a text-editor and save them back.
+
+You simply put in a query like you normally do in beets.
+
+    `beet yamleditor beatles`
+    
+    `beet yamleditor beatles -a`
+    
+    `beet yamleditor beatles -f'$title-$lyrics'`
+    
+
+
+You get a list of hits and then you can edit them. The ``yamleditor`` opens your standard text-editor with a list of your hits and for each hit a bunch of fields.
+
+Without anything specified in your ``config.yaml`` for ``yamleditor:`` you will get
+
+for items
+
+    `track-$title-$artist-$album`      
+    
+and for albums
+
+   `$album-$albumartist`            
+
+you can get fields from the cmdline by adding
+
+    `-f '$genre $added'`
+    
+or
+
+   `-e '$year $comments'`
+
+If you use ``-f '$field ...'`` you get *only* what you specified.
+
+If you use ``-e '$field ...'`` you get what you specified *extra*.
+
+If you use ``-f or -e '$_all'`` you get all the fields.
```

Comment:
What do you think about an explicit `--all` flag instead of a phantom `$_all` field?